pkgs/by-name/co/copilot-language-server/package.nix

```
@@ -37,6 +37,17 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   };
 
   npmDepsHash = "sha256-PLX/mN7xu8gMh2BkkyTncP3+rJ3nBmX+pHxl0ONXbe4=";
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    versionCheckHook
+  ];
+  buildInputs = [ stdenv.cc.cc.lib ];
+
+  doCheck = true;
+  checkFlags = [ "--version" ];
```

```suggestion
  checkFlags = "--version";
```
First time I see this attribute. Is it really needed ? Check how it is done in Typst to get some inspiration.